definitions/artifacts/api.json

```diff
@@ -55,6 +55,15 @@
             "type": "string",
             "description": "API Version"
           }
+        },
+        "aws": {
```

Comment:
do we need all three here? is it worth doing an anyOf so we can support multi cloud apps but also single cloud apps?

Reply:
I think we could do `anyOf` for multi-cloud apps. I only added all 3 so that the artifact was cloud agonostic.

Reply:
Added all three as optional so any cloud can annotate, but optional so end-user services don't have to set anything.

Reply:
@xpositivityx this was on a prev resolved comment for ref: -> https://www.jsonschemavalidator.net/s/d9Go90gx